source/administration/object-management.rst

```diff
@@ -83,23 +102,31 @@ MinIO supports both :ref:`duration based object retention <minio-object-locking-
    :align: center
    :width: 600px
 
-See :ref:`minio-object-locking` for more complete documentation.
+Delete operations against a WORM-locked object depend on the specific operation:
+
+- Delete operations which do not specify a version ID result in the creation of a "Delete Marker"
+- Delete operations which specify the version ID of a locked object result in an locking error
+
+You *must* enable object locking during bucket creation.
+Enabling bucket creation also enables :ref:`versioning <minio-bucket-versioning>`.
```

Comment:
What? This doesn't make sense.
Bucket creation does not automatically enable versioning.

Reply:
yes, i had a brain